### PR TITLE
Add complete task endpoint

### PR DESCRIPTION
**Pull request by ChrisBullOz**

### CODE REVIEW

**Review by nuno-santos-onfido — APPROVED** [review]

File: openapi.yaml [hunk 1]
```diff
@@ -85,6 +85,8 @@ paths:
     $ref: paths/workflow_runs.yaml#/tasks
   /workflow_runs/{workflow_run_id}/tasks/{task_id}:
     $ref: paths/workflow_runs.yaml#/task
+  /workflow_runs/{workflow_run_id}/tasks/{task_id}/complete:
+    $ref: paths/workflow_runs.yaml#/task
```
Comment (HoneyryderChuck) [inline openapi.yaml:89]: `/complete` is missing at the end, right?

File: output.json [hunk 2]
```diff
@@ -4434,6 +4434,56 @@
         }
       }
     },
+    "/workflow_runs/{workflow_run_id}/tasks/{task_id}/complete": {
+      "post": {
+        "summary": "Completes a Send / Receive Data task.",
```
Comment (HoneyryderChuck) [inline output.json:4439]: "Completes a task."

File: openapi.yaml (hunk 1 above)
Comment (ChrisBullOz) [reply openapi.yaml:89]: The `/complete` only applies to the endpoint URL. Line 89 is the path reference to where the endpoint is described in the workflow_runs.yaml file

**Review by HoneyryderChuck — APPROVED** [review]

File: paths/workflow_runs.yaml [hunk 3]
```diff
@@ -162,4 +162,35 @@
...
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                data:
+                  type: object
+                  description: The Task completion payload.
+      responses:
+        "204":
```
Comment (nuno-santos-onfido) [inline paths/workflow_runs.yaml:193]: Small detail sorry, forgot to mention but the return code for this was update to be a 200 with an empty body {}. This was to account for a limitation in the sdk and also to leave it ready for a future where we may want to return something as output of this call.

File: output.json [hunk 4]
```diff
@@ -4434,6 +4434,56 @@
...
+                    "type": "object",
+                    "description": "The Task completion payload."
+                  }
+                }
+              }
+            }
+          }
+        },
+        "responses": {
+          "204": {
```
Comment (nuno-santos-onfido) [inline output.json:4478]: Like mentioned in the other comment below, this is instead a 200

**Pull request merged**